Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "quickdna"
 version = "0.5.0"
 edition = "2021"
-authors = ["Theia Vogel <theia@vgel.me>"]
```

Comment:
`authors` has to be present. Otherwise poetry won't build the python package. @Ravna what sould we put here? Something like `dev@securedna.org` or similar?

Reply:
Sure, and then I can create the alias.  Unless someone has a better idea.  We could also use `quickdna@securedna.org` if we think that makes more sense.

(I'm tempted to say `example@example.com` but that would be rude to everyone. :)

Reply:
OK. Poetry is happy now. Let me know whether the new name + email is acceptable :)